frappe/installer.py

```diff
@@ -14,8 +14,8 @@
 def _new_site(
 	db_name,
 	site,
-	mariadb_root_username=None,
-	mariadb_root_password=None,
+	db_root_username=None,
```

Comment:
You'll need to change/add options for other APIs that use this and similar naming: Check commands `restore`, `reinstall` too. 

Also, the command `drop-site` seems to have inconsistent options naming...If we're using `--db-root-username` and `--db-root-password` in the aforementioned commands, we can add them there too, for consistency.